sdcm/nemesis.py

```diff
@@ -513,6 +513,25 @@ def disrupt_switch_between_PasswordAuthenticator_and_SaslauthdAuthenticator_and_
         else:
             raise UnsupportedNemesis("SaslauthdAuthenticator can't work without saslauthd environment")
 
+    def disrupt_toggle_internode_compression(self):
```

Comment:
name is misleading, toggle means on/off functionality, but in this nemesis, you randomly set compression from the ['dc', 'all', 'none'] values...
better name can be:
`random_set_compression_on_all_dbnodes()`

Reply:
done